src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,21 @@
-export const PostInfo = () => <>Put the post here</>;
+import React from 'react';
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+    </div>
+    <p className="PostInfo__body">{post.body}</p>
+    <hr />
+    <UserInfo user={post.user} />
```

Comment:
user can be undefined, so let's add the check here 
```suggestion
   {post.user && <UserInfo user={post.user} />}
```